bot/app/database/db_objects.py

```diff
@@ -230,4 +217,62 @@ def all_users(self):
         conn.close()
         curs.close()
 
-        return users
\ No newline at end of file
+        return users
+
+def get_next_event():
+
+    """Retrieves the next upcoming event from the db, returns an Event object
+    """
+
+    conn = sqlite3.connect(DB_NAME)
+    curs = conn.cursor()
+    
+    curs.execute("""select max(eventID) from Events""")
+    event_id = curs.fetchone()
+    
+    curs.execute("""select * from Events where eventID=?""", event_id)
+    next_event = curs.fetchone()
+    
+    curs.close()
+    conn.close()  
+    
+    return next_event
+
+def get_artists_event(e_name):
+
+    """Retrieves all artists playing at a certain event, returns a list
+
+    Arguments:
+        * e_name (str): name of the event to look for
+    """
+
+    conn = sqlite3.connect(DB_NAME)
+    curs = conn.cursor()
+
+    curs.execute("select aName from PlaysAt where eName=?", (e_name, ))
+    artists = curs.fetchall()
+    artists = [artist[0] for artist in artists]
+
+    curs.close()
+    conn.close()  
+
+    return artists
```

Comment:
This should be a list of `Artist` objects.

Reply:
fixed?

Reply:
see discussion below